### PR TITLE
Add frat stuff, PTS stuff, make /welcome better

### PR DESCRIPTION
**Pull request by rmichnovicz**

### CODE REVIEW

File: faq/_posts/2017-01-20-frats-and-sororities.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Check out Tech's official
+[Guide To Greek Life](http://greek.gatech.edu/uploads/files/2016GuidetoGreekLife%281%29.pdf)
+for info about social fraternities and sororities. Also see their official
+[grade and membership reports](http://greek.gatech.edu/content/4/grade-membership-reports).
+[Wikipedia](https://en.wikipedia.org/wiki/List_of_fraternities_and_sororities_at_Georgia_Institute_of_Technology)
+also maintains a list of social fraternities and sororities.
+
+There's a lot of selective organizations with Greek letters at most colleges,
+and Georgia Tech is no exception! There's 4 types of organizations I am
+referring to here: social fraternities/sororities, social
```
Comment (joshuamorton) [inline faq/_posts/2017-01-20-frats-and-sororities.md]: You double typed social sororities/fraternities.

File: faq/_posts/2017-01-20-frats-and-sororities.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+friendships. These organizations are a big deal at Georgia Tech: 25% of
+undergraduates are a member of one! Every one of these groups is a member of one
+umbrella council: Interfraternity Council (IFC) (32 fraternities), Collegiate
+Panhellenic Council (CPC) (8 sororities + 2 associate chapters with separate
+recruitment from the rest), Multicultural Greek Council (4 cultural interest
+fraternities + 3 sororities), and National Pan-Hellenic Council (NPHC)
+(4 historically black fraternities + 4 sororities).
+
+Interfraternity Council fraternities are what people think of when one mentions
+frats. Despite their often large houses, Tech's frats are typically small to
```
Comment (joshuamorton) [inline faq/_posts/2017-01-20-frats-and-sororities.md]: I'd prefer if "frats" was replaced with "fraternities" throughout, except perhaps in this first usage. They normally dislike being referred to as "frats" and respecting that seems reasonable.

**Review by joshuamorton — CHANGES_REQUESTED** [review]

File: faq/_posts/2017-01-20-frats-and-sororities.md (hunk 2 above)
Comment (copycatchiller) [reply faq/_posts/2017-01-20-frats-and-sororities.md]: Yes, I was just going to say this. I'll revise this post soon (as I am in a fraternity :P)

Comment (rmichnovicz) [reply faq/_posts/2017-01-20-frats-and-sororities.md]: kk crtl+f'd done

Comment (copycatchiller) [reply faq/_posts/2017-01-20-frats-and-sororities.md]: Overall it looks pretty complete, though. Can you replace "frats" with "fratenities" and such. I'll accept the PR then.

**Review by joshuamorton — APPROVED** [review]

**Pull request merged**